azuredeploy.json

```diff
@@ -84,7 +108,7 @@
             "apiVersion": "2016-09-01",
             "name": "[variables('hostingPlanName')]",
             "type": "Microsoft.Web/serverfarms",
-            "location": "[resourceGroup().location]",
+            "location": "[parameters('location')]",
```

Comment:
Can we retain using the resourceGroup location for other types, but enforce the limitations on only the search service?